testplugins/src/main/java/org/spongepowered/test/customdata/CustomDataTest.java

```diff
@@ -135,11 +136,12 @@ public void onRegisterSpongeCommand(final RegisterCommandEvent<Command.Parameter
                             break;
                         case USER:
                             // delegate to player
-                            this.customUserData(player.uniqueId(), number);
-                            player.kick(Component.text("Setting User data..."));
-                            final Scheduler scheduler = Sponge.server().scheduler();
-                            scheduler.submit(Task.builder().delay(Ticks.single()).execute(() -> this.customUserData(player.uniqueId(), number)).plugin(this.plugin).build());
-                            scheduler.submit(Task.builder().delay(Ticks.of(2)).execute(() -> this.customUserData(player.uniqueId(), number)).plugin(this.plugin).build());
+                            this.customUserData(player.uniqueId(), number).thenAcceptAsync(v -> {
+                                player.kick(Component.text("Setting User data..."));
+                                final Scheduler scheduler = Sponge.server().scheduler();
+                                scheduler.submit(Task.builder().delay(Ticks.single()).execute(() -> this.customUserData(player.uniqueId(), number).join()).plugin(this.plugin).build());
+                                scheduler.submit(Task.builder().delay(Ticks.of(2)).execute(() -> this.customUserData(player.uniqueId(), number).join()).plugin(this.plugin).build());
+                            }, Sponge.server().scheduler().createExecutor(this.plugin));
```

Comment:
Since this is a testplugin it's a good idea to add `.exceptionally` and showcase exception handling

Reply:
...but it's not going to call an exception here, so why?

I'm not interested in changing the test plugins in this PR, only to get them working. Once this is merged, if you want to update the plugins, have at it.

Reply:
> I'm not interested in changing the test plugins in this PR, only to get them working.

:+1: